integrations/dynatrace/CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
+
+# Port_Ocean 1.0.2 (2024-01-29)
+
+### Improvements
+
+- Formatted yaml and json files (1.0.2)
+
+
+# Port_Ocean 1.0.1 (2024-01-29)
+
+### Improvements
+
+- Implemented central method for paginated resources (1.0.1)
+- Removed entity_type from ingested objects (1.0.1)
+
+### Bug Fixes
+
+- Fixed bugs in queries (1.0.1)
+
+
+# Port_Ocean 1.0.0 (2024-01-29)
+
+### Features
+
+- Add Dynatrace integration (1.0.0)
```

Comment:
Since the integration has not been published yet, all these changes is still in version 0.1.0. So create just 1 Features changelog

`Implemented Dynatrace integration to bring slos, entities and problems data to Port`